tests/dummy/app/index.html

```diff
@@ -3,27 +3,23 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Ember Gestures</title>
-    <meta name="description" content="TouchAction and Gesture support for Ember Applications">
-    <meta name="viewport"
-          content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">
```

Comment:
need to preserve the dummy app name / meta and bootstrap style import in this file